test/functional/services/remote/webdriver.ts

```diff
@@ -64,18 +64,28 @@ async function attemptToCreateCommand(log: ToolingLog, browserType: Browsers) {
   const buildDriverInstance = async () => {
     switch (browserType) {
       case 'chrome':
-        const chromeOptions = new chrome.Options();
-        const loggingPref = new logging.Preferences();
-        loggingPref.setLevel(logging.Type.BROWSER, logging.Level.ALL);
-        chromeOptions.setLoggingPrefs(loggingPref);
+        const chromeCapabilities = Capabilities.chrome();
+        const chromeOptions = [
+          'disable-translate',
+          'new-window',
+          'no-sandbox',
+          'allow-file-access-from-files',
+          'use-fake-device-for-media-stream',
+          'use-fake-ui-for-media-stream',
+        ];
         if (process.env.TEST_BROWSER_HEADLESS) {
           // Use --disable-gpu to avoid an error from a missing Mesa library, as per
           // See: https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md
-          chromeOptions.addArguments('headless', 'disable-gpu');
+          chromeOptions.push('headless', 'disable-gpu');
         }
+        chromeCapabilities.set('goog:chromeOptions', {
+          w3c: false,
+          args: chromeOptions,
+        });
+        chromeCapabilities.set('goog:loggingPrefs', { browser: 'ALL' });
```

Comment:
Had to use capabilities instead of chromeOptions to pass `w3c` flag properly. Additional options required to run new Chromedriver on CI (Linux?), without adding it I face following errors:
```
21:08:52      │ERROR WebDriverError: unknown error: Chrome failed to start: exited abnormally
21:08:52      │        (unknown error: DevToolsActivePort file doesn't exist)
21:08:52      │        (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
```